.github/workflows/bump-golang-previous.yml

```diff
@@ -23,3 +23,4 @@ jobs:
           vaultUrl: ${{ secrets.VAULT_ADDR }}
           vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
           vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
+          command: '--experimental apply'
```

Comment:
This input doesn't seems to be present in the `./.github/actions/bump-golang` workflow.

Reply:
[`812759a` (#403)](https://github.com/elastic/golang-crossbuild/pull/403/commits/812759ab6be0eaa05620da9126aef2d4459b5c81)